flocker/cli/script.py

```diff
@@ -250,9 +250,49 @@ def _changestate_on_nodes(self, deployment, deployment_config,
         return DeferredList(results)
 
 
+@flocker_standard_options
+class CLIOptions(Options):
+    """
+    Command line options for ``flocker`` CLI.
+
+    """
+    longdesc = """flocker does nothing (yet).
+
+    """
+
+    synopsis = ("Usage: flocker [OPTIONS] "
+                "\n"
+                "If you have any issues or feedback, you can talk to us: "
```

Comment:
If we're going to keep as-is for the new CLI, absolutely, but I don't know if we're keeping this text the same or not yet, another thing I need to ask about in an upcoming engineering meeting.